xdsl/dialects/func.py

```diff
@@ -67,10 +67,11 @@ class Call(Operation):
     # TODO how do we verify that the types are correct?
 
     @staticmethod
-    def get(callee: Union[str, FlatSymbolRefAttr],
-            operands: List[Union[SSAValue, Operation]],
+    def get(callee: Union[str, FlatSymbolRefAttr], ops: List[Union[SSAValue,
+                                                                   Operation]],
             return_types: List[Attribute]) -> Call:
-        return Call.build(operands=[operands],
+        ops = [op for op in ops] if ops != () else []
```

Comment:
I think a part of the confusion of the type checker is reusing the ops name. In either case, `ops_list = list(ops)` also would work